### ISSUE TITLE
stopped: run_rl_swarm.sh: line 160: 91115 Killed: 9

### ISSUE THREAD

**Issue by sujoy668** [gensyn-ai/rl-swarm#128]
ch': 0.72}
{'loss': 0.0, 'grad_norm': 0.0012030157959088683, 'learning_rate': 3.4096741493194193e-09, 'rewards/proper_id_reward_func': 0.0, 'rewards/correctness_reward_func': 0.0, 'rewards/strict_format_reward_func': 0.0, 'rewards/soft_format_reward_func': 0.0, 'rewards/xmlcount_reward_func': 0.0, 'rewards/cumulative_reward_1': 0.0, 'reward': 0.0, 'reward_std': 0.0, 'completion_length': 264.90625, 'kl': 0.0013565562737767323, 'epoch': 0.8}
{'train_runtime': 9994.0181, 'train_samples_per_second': 0.032, 'train_steps_per_second': 0.002, 'train_loss': 1.1517145951827832e-06, 'epoch': 0.8}
100%|████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [2:46:34<00:00, 499.71s/it]
***** train metrics *****
  total_flos               =        0GF
  train_loss               =        0.0
  train_runtime            = 2:46:34.01
  train_samples            =        200
  train_samples_per_second =      0.032
  train_steps_per_second   =      0.002
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:dormant toothy bat:Saving model
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:dormant toothy bat:Model saved to runs/gsm8k/multinode/Qwen2.5-0.5B-Instruct-Gensyn-Swarm-dormant_toothy_bat
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:dormant toothy bat:Tokenizer saved to runs/gsm8k/multinode/Qwen2.5-0.5B-Instruct-Gensyn-Swarm-dormant_toothy_bat
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:dormant toothy bat:📉 Finished training round: 192 stage: 1
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:dormant toothy bat:📈 Training round: 192 stage: 2
Generating train split: 160 examples [00:00, 5265.38 examples/s]
Map: 100%|██████████████████████████████████████████████████████████████████████████████████████| 160/160 [00:00<00:00, 10445.93 examples/s]
{'loss': 0.0, 'grad_norm': 10.366936683654785, 'learning_rate': 5e-07, 'rewards/consensus_reward_func': 0.0, 'rewards/concensus_correctness_reward_func': 0.0, 'rewards/question_recreation_reward_func': 0.10419253999134526, 'rewards/final_correctness_reward_func': 0.0625, 'rewards/strict_format_reward_func': 0.0, 'rewards/soft_format_reward_func': 0.0, 'rewards/xmlcount_reward_func': 0.0, 'rewards/cumulative_reward_2': 0.0, 'reward': 0.16669253999134526, 'reward_std': 0.13655179666238837, 'completion_length': 267.46875, 'kl': -1.0628308011462195e-08, 'epoch': 0.1}
 10%|██████████                                                                                           | 2/20 [12:12<1:51:41, 372.32s/it]./run_rl_swarm.sh: line 160: 91115 Killed: 9               python -m hivemind_exp.gsm8k.train_single_gpu --hf_token "$HUGGINGFACE_ACCESS_TOKEN" --identity_path "$IDENTITY_PATH" --modal_org_id "$ORG_ID" --config "$CONFIG_PATH"

**Comment by h-grieve** [comment]
What hardware are you running? @sujoy668
